test/contracts/cairo/contract.cairo

```diff
@@ -1,4 +1,4 @@
-// Declare this file as a StarkNet contract and set the required
+// Declare this file as a Starknet contract and set the required
```

Comment:
Changing the content of smart contract files might change some expected addresses and hashes, so I'd leave them intact.